applications/GeoMechanicsApplication/custom_elements/stress_state_policy.h

```diff
@@ -40,23 +40,34 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) StressStatePolicy
 protected:
     static const Vector VoigtVector2D;
     static const Vector VoigtVector3D;
+    static const Vector VoigtVectorInterface2D;
 
     static constexpr SizeType GetVoigtSize2D() { return VOIGT_SIZE_2D_PLANE_STRAIN; }
 
+    static constexpr SizeType GetVoigtSizeInterface2D() { return VOIGT_SIZE_2D_INTERFACE; }
+
     static constexpr SizeType GetVoigtSize3D() { return VOIGT_SIZE_3D; }
 
+    static constexpr SizeType GetVoigtSizeInterface3D() { return VOIGT_SIZE_3D_INTERFACE; }
+
     static constexpr SizeType GetStressTensorSize2D() { return STRESS_TENSOR_SIZE_2D; }
 
     static constexpr SizeType GetStressTensorSize3D() { return STRESS_TENSOR_SIZE_3D; }
 
 private:
     static Vector DefineVoigtVector(std::size_t Dimension);
+    static Vector DefineInterfaceVoigtVector(std::size_t Dimension);
 
     static constexpr std::size_t GetVoigtSize(std::size_t Dimension)
     {
         return Dimension == N_DIM_3D ? GetVoigtSize3D() : GetVoigtSize2D();
     }
 
+    static constexpr std::size_t GetInterfaceVoigtSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? GetVoigtSizeInterface3D() : GetVoigtSizeInterface2D();
```

Comment:
This already anticipates 3D whilst we are first trying to accomplish 2D plane strain interfaces.

Reply:
Removed